modules/BackToStart/src/backtostart.cpp

```diff
@@ -10,7 +10,9 @@ BackToStart::BackToStart() : Module(BackToStart::moduleName)
 {
 	getObjectReturnTrajectoryService = create_service<atos_interfaces::srv::GetObjectReturnTrajectory>(ServiceNames::getObjectReturnTrajectory,
 		std::bind(&BackToStart::onReturnTrajectoryRequest, this, _1, _2));
-	this->declare_parameter("turnRadius", 5.0);
+	this->declare_parameter("turn_radius", 5.0);
+	this->declare_parameter("min_speed", 4.0);
+	this->declare_parameter("max_speed", 12.0);
```

Comment:
An idea: Variables would be added e.g. 
default_turn_radius = 5.0;
instead of 5.0 in the declare_parameter method.

Reply:
Hmm, it seems a bit redundant since the input itself is defined as default value. I have never seen a ros default parameter value be set with a variable before